livekit-ffi/src/server/requests.rs

```diff
@@ -470,19 +472,19 @@ unsafe fn on_to_argb(
             match rgba_format {
                 proto::VideoFormatType::FormatArgb => {
                     #[rustfmt::skip]
-                    yuv_helper::i420_to_argb(src_y, yuv.stride_y, src_u, yuv.stride_u, src_v, yuv.stride_v, argb, stride, w, h);
+                    yuv_helper::i420_to_bgra(src_y, yuv.stride_y, src_u, yuv.stride_u, src_v, yuv.stride_v, argb, stride, w, h);
```

Comment:
I'm not understanding this, isn't this mismatched from the indicated format?

Reply:
FourCC are defined in BigEndian https://en.wikipedia.org/wiki/FourCC#Compiler_support (It is just the naming convention)
So on little endian system ARGB is indeed BGRA in memory

Reply:
Also more details here https://chromium.googlesource.com/libyuv/libyuv/+/refs/heads/main/docs/formats.md#the-argb-fourcc